doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -295,7 +295,285 @@ Your output should look like the following:
+
+::
+
+    [_]  eksctl version 0.10.1
+    [_]  using region us-east-1
+    [_]  IAM OpenID Connect provider is associated with cluster "my-cluster" in "us-east-1"
+
+Now that the cluster has an OIDC identity provider, you can create a
+role and give a Kubernetes ServiceAccount permission to assume the role.
+
+Get your OIDC ID
+^^^^^^^^^^^^^^^^
+
+To set up the ServiceAccount, first obtain the OpenID Connect issuer URL
+using the following command:
+
+::
+
+    aws eks describe-cluster --name ${CLUSTER_NAME} --region ${AWS_REGION} \
+        --query cluster.identity.oidc.issuer --output text
+
+The command will return a URL like the following:
+
+::
+
+    https://oidc.eks.${AWS_REGION}.amazonaws.com/id/D48675832CA65BD10A532F597OIDCID
+
+In this URL, the value D48675832CA65BD10A532F597OIDCID is the OIDC ID.
+The OIDC ID for your cluster will be different. You need this OIDC ID
+value to create a role.
+
+If your output is \ ``None``, it means that your client version is old.
+To work around this, run the following command: 
+
+::
+
+    aws eks describe-cluster --query cluster --name ${CLUSTER_NAME} --output text | grep OIDC
+
+The OIDC URL will be returned as follows:
+
+::
+
+    OIDC https://oidc.eks.us-east-1.amazonaws.com/id/D48675832CA65BD10A532F597OIDCID
+
+Create your IAM Role 
+^^^^^^^^^^^^^^^^^^^^
+
+Create a file named \ ``trust.json``  and insert the following trust
+relationship code block into it. Be sure to replace all \ ``<OIDC ID>``, \ ``<AWS account number>``, and \ ``<EKS Cluster region>`` placeholders with values corresponding to your cluster.
+
+::
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Principal": {             "Federated": "arn:aws:iam::<AWS account number>:oidc-provider/oidc.eks.<EKS Cluster region>.amazonaws.com/id/<OIDC ID>"
+          },
+          "Action": "sts:AssumeRoleWithWebIdentity",
+          "Condition": {
+            "StringEquals": {
+              "oidc.eks.<EKS Cluster region>.amazonaws.com/id/<OIDC ID>:aud": "sts.amazonaws.com",               "oidc.eks.<EKS Cluster region>.amazonaws.com/id/<OIDC ID>:sub": "system:serviceaccount:<Namespace>:sagemaker-k8s-operator-default"
+            }
+          }
+        }
+      ]
+    }
+
+Run the following command to create a role with the trust
+relationship defined in \ ``trust.json``. This role enables the
+Amazon EKS cluster to get and refresh credentials from IAM.
+
+::
+
+    aws iam create-role --role-name <role name> --assume-role-policy-document file://trust.json --output=text
+
+Your output should look like the following:
+
+::
+
+    ROLE    arn:aws:iam::123456789012:role/my-role 2019-11-22T21:46:10Z    /       ABCDEFSFODNN7EXAMPLE   my-role
+    ASSUMEROLEPOLICYDOCUMENT        2012-10-17
+    STATEMENT       sts:AssumeRoleWithWebIdentity   Allow
+    STRINGEQUALS    sts.amazonaws.com       system:serviceaccount:my-namespace:sagemaker-k8s-operator-default
```

Comment:
Can we call out somewhere that `my-namespace` is the one this guide is using for the namespaced deployment.